### PR TITLE
[skip ci] Add docker scripts for use in the README tutorial.

### LINKED ISSUE #151: Add a Dockerfile for easier deployment
@jfstephe I think this is what you may be looking for

### PR DESCRIPTION
**Pull request by amcp**
@jfstephe FYI, I've added Docker to the README to simplify the first five minutes even further. Let me know what you think.
Resolves #151

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,93 +39,75 @@ The graph has a vertex per comic book character with an edge to each of the
...
-    cd server/dynamodb-janusgraph010-storage-backend-1.0.0
-    ```
-7. Start Gremlin Server with the DynamoDB Local configuration.
-
-    ```bash
-    bin/gremlin-server.sh ${PWD}/conf/gremlin-server/gremlin-server-local.yaml
-    ```
-8. Start a Gremlin shell with `bin/gremlin.sh` and connect to the Gremlin Server
-endpoint. Always execute commands on the server
+3. After the Gremlin shell starts, set it up toexecute commands remotely.
```
Comment (amcp) [inline README.md]: toexecute -> to execute

File: README.md [hunk 2]
```diff
@@ -39,93 +39,75 @@ The graph has a vertex per comic book character with an edge to each of the
 comic books in which they appeared.
 
 ### Load a subset of the Marvel Universe Social Graph
-1. Install prerequisites and clone the repository in GitHub.
+1. Install the prerequisites (Git, JDK 1.8, Maven, Docker) of this tutorial.
+The command below uses a
+[convenience script for Amazon Linux](https://raw.githubusercontent.com/awslabs/dynamodb-janusgraph-storage-backend/master/src/test/resources/install-reqs.sh)
+on EC2 instances that installs Git, Open JDK 1.8, Maven, and Docker. It adds the ec2-user to the docker group so that you can
```
Comment (amcp) [inline README.md]: that installs -> to install

File: src/test/resources/dynamodb-janusgraph-docker/wait-for-dynamodb-local.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (amcp) [inline src/test/resources/dynamodb-janusgraph-docker/wait-for-dynamodb-local.sh]: remove this file

File: src/test/resources/dynamodb-local-docker.properties [hunk 4]
```diff
@@ -0,0 +1,103 @@
+#
```
Comment (amcp) [inline src/test/resources/dynamodb-local-docker.properties]: create an issue to streamline the .properties files

Comment (amcp) [inline src/test/resources/dynamodb-local-docker.properties]: #212

**Comment by sjudeng** [comment]
@amcp Yeah the service healthcheck in compose looks great. This could replace the wait-for-es/etc. script(s) we currently have.

**Pull request merged**